setup.cfg

```diff
@@ -12,8 +12,8 @@ all_files  = 1
 [upload_sphinx]
 upload-dir = docs/build/html
 
-[pep8]
-ignore = E221
+[pycodestyle]
```

Comment:
For development I believe it is fine to demand an uptodate version, but its your call. Thanks for merging the commit.